pkg/backup/generative_split_and_scatter_processor.go

```diff
@@ -554,13 +567,7 @@ func runGenerativeSplitAndScatter(
 				// cluster.
 				for importSpanChunk := range restoreEntryChunksCh {
 					scatterKey := importSpanChunk.entries[0].Span.Key
-					if !importSpanChunk.splitKey.Equal(roachpb.Key{}) {
```

Comment:
Is splitKey used anymore?

I think there are comments above at lines 555-559 that need to be updated.

Reply:
Done.